src/PrestaShopBundle/Resources/config/services/bundle/form/form_type.yml

```diff
@@ -1759,6 +1758,22 @@ services:
     tags:
       - { name: form.type }
 
+  form.type.sell.product.combination.combination_images_choice_type:
```

Comment:
Just a detail but could you please put this definition a bit upper? Around line 1727

It's just so that single form definitions are grouped together at one place, and bulk forms at another And not all of them mixed together